### PR TITLE
Update quota descriptions

### PR DESCRIPTION
**Pull request by arolshan**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [x] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [x] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [x] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json [hunk 1]
```diff
@@ -423,7 +423,7 @@
             "values": [
               {
                 "value": "RespectQuota",
-                "description": "Ingestion enabled following daily cap quota reset, or subscription enablement."
+                "description": "Renewed ingestion following daily cap quota reset, or subscription activation."
```
Comment (yoramsinger) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:426]: It's not renewed, it's the stable state where workspace has not ingested more than the quota assigned.
"Daily quota has not been reached."

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json [hunk 2]
```diff
@@ -435,15 +435,15 @@
               },
               {
                 "value": "OverQuota",
-                "description": "Reached daily cap quota, ingestion stopped."
+                "description": "Stopped ingestion following daily cap quota reach."
```
Comment (yoramsinger) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:438]: keep it simple: "Daily quota has been reached and data ingestion was stopped."

**Review by mmyyrroonn — APPROVED** [review]

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json [hunk 3]
```diff
@@ -435,15 +435,15 @@
...
               },
               {
                 "value": "SubscriptionSuspended",
-                "description": "Ingestion stopped following suspended subscription."
+                "description": "Stopped ingestion following subscription getting into suspended state."
```
Comment (yoramsinger) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:442]: The subscription is suspended and ingestion was stopped.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json [hunk 4]
```diff
@@ -435,15 +435,15 @@
...
               },
               {
                 "value": "ApproachingQuota",
-                "description": "80% of daily cap quota reached."
+                "description": "80% of the daily cap quota reached."
```
Comment (yoramsinger) [inline specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:446]: Remove cap. "80% of the daily quota reached."

Comment (yossi-y) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:446]: You don't want to say that as users can have payment considerations for daily cap.
"80% of the daily cap quota reached. Consider raising your daily cap limit and not loose any data"

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json (hunk 3 above)
Comment (yossi-y) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:442]: "Stop data ingestion following suspended subscription"

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json (hunk 2 above)
Comment (yossi-y) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:438]: "Stop data ingestion following daily cap quota reach"

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json (hunk 1 above)
Comment (yossi-y) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:426]: In this case:
"Renewed data ingestion following daily cap quota reset, or subscription enablement."

Comment (yoramsinger) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:426]: I don't think we need to specify what could have happened here. This status specifies a steady state where data is ingested and quota has not been exceeded. Why complicate with renew, daily cap reset, subscription enablement and all those fancy words.

File: specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json (hunk 2 above)
Comment (yoramsinger) [reply specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/preview/2020-03-01-preview/Workspaces.json:438]: I like mine better. I don't think we should use cap here. It's daily quota or daily cap. not both. And cap is kind of a word that has no dictionary entry. I prefer not to use it anyhow.

**Comment by pilor** [comment]
No API changes here, no ARM review required

**Comment by MyronFanQiu** [comment]
@arolshan Hi. Do we need this PR or not?

**Comment by arolshan** [comment]
Closed, thanks

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**